lib/modules/showImages.js

```diff
@@ -1268,7 +1266,48 @@ function generateIframe(options) {
 	element.independent = true;
 
 	keepMediaVisible(iframeNode);
-	makeMediaIndependentOnResize(element, iframeNode);
+	makeMediaIndependentOnResize(element, element);
```

Comment:
I see that my earlier comment regarding this was wrong. For `makeMediaIndependentOnResize` to work, the 2nd element must be in the subtree of `element`.